core/templates/dev/head/services/rte-helper.service.ts

```diff
@@ -75,7 +75,7 @@ angular.module('oppia').factory('RteHelperService', [
         return angular.copy(_RICH_TEXT_COMPONENTS);
       },
       isInlineComponent: function(richTextComponent) {
-        var inlineComponents = ['link', 'math', 'concept-card'];
```

Comment:
Was this wrong before? (I don't see "concept-card" anywhere else.)

If so how do we prevent skew from happening again?

Reply:
Yeah, it was. 
I could add a check below this line to verify that all of these are indeed frontend IDs of valid RTE components. Should I do that?

Reply:
Probably better to do that in tests than in the main code. 

How about this: extract inlineComponents into an INLINE_RTE_COMPONENTS constant and write a unit test to ensure that all entries in that list are valid?

Reply:
Done, added a test to `rte_component_registry_test`.

Reply:
Done, added a test to `rte_component_registry_test`.